getter.go

```diff
@@ -144,6 +144,7 @@ func (g *getter) queueFile(url *url.URL) (http.Header, error) {
 	// resp could be nil, depending on the error.
 	if err != nil {
 		g.chunkWg.Done()
+		defer resp.Body.Close()
```

Comment:
ahh because we fire off the `initChunks()` asynchronously the current goroutine will close out and call the defer.  where is the body close then for when `initChunks()` does it work?  not seeing that inside `initChunks()` and we only return the `resp.Header` so it seems the `resp` is hanging out in that case